drawee/src/main/java/com/facebook/drawee/drawable/RoundedCornersDrawable.java

```diff
@@ -162,17 +205,20 @@ private void updatePath() {
   @Override
   public void draw(Canvas canvas) {
     Rect bounds = getBounds();
+    int saveCount = canvas.save();
     switch (mType) {
       case CLIPPING:
         // clip, note: doesn't support anti-aliasing
-        int saveCount = canvas.save();
         mPath.setFillType(Path.FillType.EVEN_ODD);
         canvas.clipPath(mPath);
         super.draw(canvas);
         canvas.restoreToCount(saveCount);
         break;
       case OVERLAY_COLOR:
+        mPath.setFillType(Path.FillType.EVEN_ODD);
+        canvas.clipPath(mPath);
```

Comment:
Don't use `clipPath`. This is not always implemented and it would cause a crash on a lot of devices.